<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gammaray::TextDocumentInspector</class>
 <widget class="QWidget" name="Gammaray::TextDocumentInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <widget class="QSplitter" name="splitter_13">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_1">
           <layout class="QVBoxLayout" name="verticalLayout_17">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>All documents:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeView" name="documentList">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <property name="uniformRowHeights">
               <bool>true</bool>
              </property>
              <property name="allColumnsShowFocus">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QSplitter" name="splitter_12">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Document structure:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeView" name="documentTree">
               <property name="uniformRowHeights">
                <bool>true</bool>
               </property>
               <property name="allColumnsShowFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget_3">
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Element format:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTreeView" name="documentFormatView">
               <property name="rootIsDecorated">
                <bool>false</bool>
               </property>
               <property name="uniformRowHeights">
                <bool>true</bool>
               </property>
               <property name="allColumnsShowFocus">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QTextEdit" name="documentView"/>
         </widget>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
